.travis.yml

```
@@ -13,7 +13,7 @@ jobs:
     - stage: test
       name: Build on OpenJDK 8
       jdk: openjdk8
-      script: ./gradlew --no-daemon -i --stacktrace jacocoTestReport
+      script: ./gradlew --no-daemon -i -Dtests.integration=localstack --stacktrace jacocoTestReport
```

```suggestion
      script: ./gradlew --no-daemon -i --stacktrace jacocoTestReport
```

Back to the "non-integration" tests